java/build.gradle

```diff
@@ -12,6 +12,17 @@ repositories {
 
 subprojects {
     repositories {
+        // Runs for GITHUB_ACTIONS for testing release Maven deployment.
+        if(System.getenv("RUN_DEPLOYMENT_TEST")) {
```

Comment:
alternatively, you could get for `ORG_GRADLE_PROJECT_centralManualTestingAuthHeaderName` and `ORG_GRADLE_PROJECT_centralManualTestingAuthHeaderValue`